src/styles/_deps.scss

```diff
@@ -0,0 +1,20 @@
+%button-reset {
```

Comment:
I started this file for us to put shared SCSS code (mixins, extends, etc)... not sure what this will end up looking like so I left it kind of generic for now 🤷‍♀️